lib/request/delete.js

```diff
@@ -95,6 +95,9 @@ module.exports = function (context) {
 
         for (k = 0, l = links.length; k < l; k++) {
           field = links[k]
+
+          if (!record[field]) continue
```

Comment:
This will fail on falsy values like `false`, `""`, `0`.

Reply:
You are correct. But do we have to create update for these kind of fields? I might not see the "big picture" here :)